package.json

```diff
@@ -17,16 +17,15 @@
     "@vitejs/plugin-react": "^4.0.3",
     "lint": "^0.8.19",
     "pretty": "^2.0.0",
-    "axios": "^1.6.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-query": "^3.39.3",
```

Comment:
why have you removed axios and react-query from `dependencies` and reinstalled in `devDependencies` ?